nixos/modules/services/misc/gitea.nix

```
@@ -421,15 +436,27 @@ in
         ROOT = cfg.repositoryRoot;
       };
 
-      server = mkIf cfg.lfs.enable {
-        LFS_START_SERVER = true;
-        LFS_JWT_SECRET = "#lfsjwtsecret#";
-      };
+      server = mkMerge [
+        (mkIf cfg.configureNginx {
+          STATIC_URL_PREFIX = lib.mkDefault "/static";
+        })
+        (mkIf cfg.lfs.enable {
+          LFS_START_SERVER = true;
+          LFS_JWT_SECRET = "#lfsjwtsecret#";
+        })
+      ];
 
       session = {
         COOKIE_NAME = lib.mkDefault "session";
       };
 
+      picture = mkIf cfg.configureNginx {
+        AVATAR_STORAGE_TYPE = lib.mkDefault "local";
+        AVATAR_UPLOAD_PATH = lib.mkDefault "data/avatars";
+        REPOSITORY_AVATAR_STORAGE_TYPE = lib.mkDefault "local";
+        REPOSITORY_AVATAR_UPLOAD_PATH = lib.mkDefault "data/repo-avatars";
```

```suggestion
        AVATAR_UPLOAD_PATH = lib.mkDefault "avatars";
        REPOSITORY_AVATAR_STORAGE_TYPE = lib.mkDefault "local";
        REPOSITORY_AVATAR_UPLOAD_PATH = lib.mkDefault "repo-avatars";
```
I am on gitea 1.21 and before I had actually twice data in the final path eg `/var/lib/gitea/data/data/repo-avatars/43`

---

Thanks, fixed!